src/entrypoint/graphql.ts

```diff
@@ -376,11 +377,17 @@ export async function startApolloServer() {
   app.use(
     expressJwt({
       secret: process.env.JWT_SECRET,
-      algorithms: ["HS256"],
+      algorithms: ["RS256"],
       credentialsRequired: false,
       requestProperty: 'token',
     }))
 
+  // JWKS endpoint to allow clients to verify jwt tokens
+  app.get("/.well-known/jwks.json", async function(req, res) {
```

Comment:
is the `.well-known` the standard way to do it? all of our API interaction currently rely on graphql, so I think that would make sense to provide the public key through the same API.

Reply:
There are many systems I know that rely on a rest api to query the public key. This would probably enable compatibility with 3rd party auth libraries. The path isn't standard but its what `auth0` currently exposes.

Reply:
let me know which one you want...

Reply:
on the mobile side we use apollo-client. maybe the easiest would be see if a particular option would have a smoother integration.